packages/cw-derive/examples/basic.rs

```diff
@@ -0,0 +1,30 @@
+#![allow(dead_code)]
+use cosmwasm_std::Response;
+
+struct Ctx;
+struct Error;
+
+#[derive(serde::Serialize, serde::Deserialize, Clone, Debug, PartialEq, schemars::JsonSchema)]
+struct Member;
+
+#[cw_derive::interface(module=msg, exec=Execute, query=Query)]
+trait Cw4 {
```

Comment:
`pub trait`

Reply:
True. Doesn't matter for example, but true.